LEAF_Request_Portal/sources/Shortener.php

```diff
@@ -54,6 +54,20 @@ private function decodeShortUID($in)
         return $res - $this->offset;
     }
 
+    public function decodeEntitiesForExcel($formQuery) {
+        foreach ($formQuery as $recordID => $record) {
+            $formQuery[$recordID]['title'] = html_entity_decode($record['title'], ENT_QUOTES);
+            if (isset($record['s1'])) {
```

Comment:
Do we need to verify that this is an array here?  I.e. not a supercalifragulistic string?

Reply:
Added param and return value types.  I also updated the method name since other programs besides excel could use this.